perfkitbenchmarker/vm_util.py

```diff
@@ -87,8 +90,15 @@ def PrependTempDir(file_name):
 
 def GenTempDir():
   """Creates the tmp dir for the current run if it does not already exist."""
-  create_cmd = ['mkdir', '-p', GetTempDir()]
-  create_process = subprocess.Popen(create_cmd)
+  if os.name == WINDOWS:
```

Comment:
Consider using python functions here. `os.path.exists` and [`os.makedirs`](https://docs.python.org/2/library/os.html#os.makedirs) work across platforms.